android/composeui/src/main/java/com/stadiamaps/ferrostar/composeui/views/InstructionsView.kt

```diff
@@ -36,23 +49,59 @@ fun InstructionsView(
     distanceToNextManeuver: Double?,
     distanceFormatter: DistanceFormatter = LocalizedDistanceFormatter(),
     theme: InstructionRowTheme = DefaultInstructionRowTheme,
-    content: @Composable () -> Unit = {
-      ManeuverImage(instructions.primaryContent, tint = MaterialTheme.colorScheme.primary)
+    remainingSteps: List<RouteStep>? = null,
+    contentBuilder: @Composable (VisualInstruction) -> Unit = {
+      ManeuverImage(it.primaryContent, tint = MaterialTheme.colorScheme.primary)
     }
 ) {
-  Column(
+  var isExpanded by remember { mutableStateOf(false) }
+  val screenHeight = LocalConfiguration.current.screenHeightDp
+  val collapsedHeight = 100.dp
```

Comment:
See lines 62 and 70. Is there a better way to do this? We need a "default" value it seems, but there's probably a better way.

Reply:
If we want to get the actual height, this looks likely: https://stackoverflow.com/questions/66752369/get-height-of-element-jetpack-compose. That said, if we can pre-set it to a known value that's actually correct with and without various extra data (lanes, secondary instruction, etc). Then that could be fine. Less work for the UI to do if it's a static value (or set of).

Reply:
Nevermind; I found a way to do this using `heightIn` and `animateContentSize`; it's not really possible to do this correctly with a fixed collaped size.